lib/tasks/demo_data_for_development.rake

```diff
@@ -47,13 +52,8 @@ namespace :data do
     conference.update_attributes!(end_date: 1.day.ago)
     conference.registration_period.update_attributes!(start_date: 9.days.ago, end_date: 8.days.ago)
 
-    # This is a full conference demo instance that will happen in the future.
-    # By full conference it means all basic information about conference is already set.
-    conference = create(:full_conference, title: 'Open Source Event Manager Demo Conference', short_title: 'osemdemo', start_date: 2.days.from_now, end_date: 6.days.from_now, start_hour: 8, end_hour: 20, description: 'This is a full conference demo instance happening in the future. It contains open cfp, venue/rooms, submitted talks by multiple speakers, partly confirmed talks and multiple schedules.')
-    generate_program conference
-
     # This is a conference demo instance that will happen in the future.
-    # No information about this conference is set by default.
+    # Basic information (like venue,room,schedule,etc) is not set by default.
```

Comment:
What we care about is what is set, not what is not set. 
Also the 'by default' part is not necessary. True if you want rooms, you have to create them, they are not set by default. But how does that information help us here? We need to know that only X,Y things are set (and no other information). Think something along those lines:
```
This is a conference that will happen in the future
It only has a registration period and unscheduled events
```